### PR TITLE
Make support for '--update' and '--watch' options

### PR DESCRIPTION
**Pull request by axard**
Hello. I'm confused and need some advice. 
- Is it normal to make PR if someone already done it?
- Should i prefer NodeJS 'fs' instead of 'chokidar', because the last is external dependency? I'm not familiar with JS ecosystem and read in readme of chokidar, that it has better support on MacOS

### CODE REVIEW

**Comment by ai** [comment]
@axard Why did you choose `chokidar`? What other alternatives we have? What is the difference between them?

File: update-snapshots.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+let findSnapshotTests = require('./find-snapshot-tests')
+
+module.exports = async function updateSnapshots (cwd) {
+  let tests = await findSnapshotTests(cwd)
+
+  let args = ['jest', '-u'].concat(tests)
+
+  let child = childProcess.spawn('npx', args)
+
+  child.stdout.pipe(process.stdout)
```
Comment (ai) [inline update-snapshots.js:12]: Does it keep `jest` color output?

File: watch-and-update-snapshots.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+let chokidar = require('chokidar')
+let childProcess = require('child_process')
+
+let findSnapshotTests = require('./find-snapshot-tests')
+
+module.exports = async function watchAndUpdateSnaphots (cwd) {
+  let tests = await findSnapshotTests(cwd)
+
+  let watcher = chokidar.watch(tests, {
```
Comment (ai) [inline watch-and-update-snapshots.js:9]: User may change the JS file, not test

**Comment by ai** [comment]
The developer of another PR fixed comment first.

But your initial attempt was really great. Nice work, just a little later.

What is your Twitter name? I will mention you on Twitter.

**Comment by axard** [comment]
Damn)) I overslept all) But i really fresh now))
Thank you for review, i see now way to level up and how i should think about choosing dependencies.
I have no Twitter.

**Pull request closed without merging**